### ISSUE TITLE
make updating less confusing

### ISSUE THREAD

**Issue by antonkulaga** [rkimoakbioinformatics/oakvar#32]
There was a bug that you fixed https://github.com/rkimoakbioinformatics/oakvar/issues/30 where excelreporter was broken. While updating to fix the bug I discovered a usability problem with the update. When I do:
```
pip install -U oakvar
```
 it unloggs me as a user from oakvar store. 
Then I usually do 
```ov module update```
However, it does not update part of the dependencies, the only way to update it is to:
```
ov store fetch
```
at the same time I am already unlogged, of course it tells that I should log in but I usually execute this command automatically without checking stuff. It took me a while (an multiple failed attempts where excelreporter was not updated at all) to understand that to update excelreporter I need to run:
```
ov system setup
ov store fetch
```
first command to get myself logged in and second to get the appstores dependencies updates.
Overall the whole setup looks really confusing to me. For example, why ov store fetch is needed at all. It is more natural just to do ov module update to get all updates that are needed.

**Comment by rkimoakbioinformatics** [comment]
@antonkulaga Thanks for letting me know. Regarding `pip install` and logging out from the store, I do not see that behavior. What do you see when you test the following?
```
ov store account login # this will prompt to log in to the store.
pip install -U oakvar # this should install v2.5.45 which will fetch the store cache before module update.
ov store account check # this will check if you are still logged in at the store.
ov module update # this should fetch the store cache and then perform module update
```

**Comment by antonkulaga** [comment]
The pip discovered 2.5.45 only now, before that I was using 2.5.44 that had the issue of not fetching

**Comment by rkimoakbioinformatics** [comment]
I released oakvar 2.5.45 after reading your post. `ov module install` had been fetching but `ov module update` did not in 2.5.44.


I wonder how logging out happened with `pip install -U oakvar`, because it should not happen. Thus, the login and check commands to see what is going on.

**Comment by antonkulaga** [comment]
I think it works fine now
